appinventor/appengine/src/com/google/appinventor/client/style/neo/ImagesNeo.java

```diff
@@ -0,0 +1,938 @@
+package com.google.appinventor.client.style.neo;
+
+import com.google.appinventor.client.Images;
+import com.google.appinventor.client.Ode;
+import com.google.gwt.resources.client.ImageResource;
+
+public interface ImagesNeo extends Images {
+
+  /**
+   * Codi the Bee for the No Projects Dialog
+   * {@link Ode#createWelcomeDialog(boolean)}
+   */
+  @Override
+  @Source("images/codi_vert.png")
+  ImageResource codiVert();
+
+  /**
+   * App Inventor Logo 
+   * {@link Ode#createWelcomeDialog(boolean)}
+   */
+  @Override
+  @Source("images/appinvlogo-32.png")
+  ImageResource appInventorLogo();
```

Comment:
I don't think this is used any more.

Reply:
> I don't think this is used any more.

Since it wasn't replaced in Neo, I removed it from there. I don't know if this PR is the place to remove it from Classic.